documentation/ame-guide/modules/ROOT/pages/modeling/abstract-entity.adoc

```diff
@@ -3,22 +3,22 @@
 
 SAMM supports defining abstract elements (entities and properties) and the concept of inheritance in order to make semantic information reusable.
 The editor allows to create abstract entities and properties and to declare relationships to inheritance via "extends".
-See https://eclipse-esmf.github.io/samm-specification/2.1.0/modeling-guidelines.html#declaring-entities[Abstract Entities]
+See https://eclipse-esmf.github.io/samm-specification/2.1.0/modeling-guidelines.html#declaring-entities[Declaring Entities]
 for more information about abstract entities and properties.
 
 There are two different ways to create an abstract entity:
 
-* Use drag-and-drop from the Workspace sidebar
-* Adjust an Entity in the extends field
+* Use drag-and-drop from the Workspace sidebar.
+* Adjust an Entity in the extends field.
 
 == Use drag-and-drop from the Workspace sidebar
 
 To add an abstract entity, proceed as follows:
 
-. Open the Workspace sidebar
-. Drag the Abstract Entity element and drop it to the graph area canvas
+. Open the Workspace sidebar.
+. Drag the Abstract Entity element and drop it to the graph area canvas.
```

Comment:
suggestion: Drag-and-drop the Abstract Entity element into the graph area canvas. (use pattern for all)